using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -587,32 +663,42 @@ ACPIdentity.getIdentifiers { (retrievedVisitorIds) in
 {% tab title="React Native" %}
 #### JavaScript
 
-### getIdentifiers
+### getIdentifiers<a id="getIdentifiers-js"></a>
+
+This `getIdentifiers` API returns all customer identifiers that were previously synced with the Adobe Experience Cloud
+
+#### Syntax
+
+```jsx
+ACPIdentity.getIdentifiers()
+```
 
-Returns all customer identifiers which were previously synced with the Adobe Experience Cloud.
+#### Example
 
 ```jsx
 ACPIdentity.getIdentifiers().then(identifiers => console.log("AdobeExperienceSDK: Identifiers = " + identifiers));
 ```
 {% endtab %}
 {% endtabs %}
 
-## Get Experience Cloud IDs
+## Get Experience Cloud IDs<a id="getExperienceCloudIdTitle"></a>
 
 {% tabs %}
 {% tab title="Android" %}
-### getExperienceCloudId
+### getExperienceCloudId<a id="getExperienceCloudId-java"></a>
 
-Retrieves the Experience Cloud ID from the Experience Cloud ID Service.
+This `getExperienceCloudId` API retrieves the Experience Cloud ID that was generated at initial launch  and stored in the Experience Cloud ID Service.
 
-The Experience Cloud ID is generated at initial launch and is stored and used from that point forward. This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall. The values are returned via the [AdobeCallback](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#adobecallback).
+This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall. The values are returned via the [AdobeCallback](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#adobecallback).
 
 #### **Syntax**
 
 ```java
 public static void getExperienceCloudId(final AdobeCallback<String> callback);
 ```
 
+o  *callback* method  will be invoked once Experience Cloud ID is available
```

Comment:
- Move to after the code example.
- Update to "*callback* is invoked after the Experience Cloud ID is available."